lib/backend/postgres/streams.ts

```diff
@@ -299,11 +299,12 @@ export class Stream extends EventEmitter {
 				this.cardTypes = _.compact(deversionedTypes);
 			}
 		}
+
 		this.streamQuery = Context.prepareQuery(
 			backend.compileSchema(this.context, this.streamer.table, select, schema, {
 				...options,
-				limit: 1,
-				extraFilter: `${this.streamer.table}.id = $1`,
+				limit: backend.MAXIMUM_QUERY_LIMIT,
```

Comment:
Why do we need the larger limit on the query?

Reply:
@LucianBuzzo there may be multiple root contracts that match the same linked contract. Which reminds me that I didn't modify the code to properly handle the case of >1 root contracts